python/llm/setup.py

```diff
@@ -334,6 +345,7 @@ def setup_package():
                         "xpu": xpu_requires,  # default to ipex 2.1 for linux and windows
                         "npu": npu_requires,
                         "xpu-2-1": xpu_21_requires,
+                        "xpu-2-3": xpu_23_requires,
```

Comment:
I used `xpu_2.3` as option here as `ipex 2.3` installation does not have device distinction for Linux. `xpu_2.3` will be better for us to support Linux with Torch 2.3 later.

@jason-dai Please let me know any suggestion on the option name. Thank you :)

Reply:
Let's use `xpu_lnl`

Reply:
Sure, have updated. Please have a look again. Thank you :)